invoices_app/factory_boy.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+import factory
+
+from invoices_app.models import (
+    InvoiceArg,
+)
+
+
+class InvoiceArgentinaFactory(factory.django.DjangoModelFactory):
```

Comment:
Could we consider having a generic `InvoiceFactory` to handle generic fields, and then `InvoiceArgentinaFactory` inherit from it?

Reply:
Resolved